src/python/dxpy/bindings/mount_all_inputs.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2014-2020 DNAnexus, Inc.
+#
+# This file is part of dx-toolkit (DNAnexus platform client libraries).
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#   use this file except in compliance with the License. You may obtain a copy
+#   of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+from __future__ import print_function
+import subprocess
+import json
+import os
+import dxpy
+from dxpy.utils import file_load_utils
+
+def _build_mount_manifest(to_mount):
+    files_list = []
+    for file_rec in to_mount:
+        file_entry = {}
+        file_handler = file_rec['handler']
+        file_entry['proj_id'] = file_handler.get_proj_id()
+        file_entry['file_id'] = file_rec['src_file_id']
+        file_name = file_rec['trg_fname']
+        k = file_name.rfind("/")
+        file_parent = file_name[:k]
+        file_entry['parent'] = "/in/" + file_parent
+        files_list.append(file_entry)
+    files_manifest = {}
+    files_manifest['Files'] = files_list
+    print("File mount manifest:")
+    print(files_manifest)
```

Comment:
Can you please give an example of this for the csv loader with the etl config file specified?